packages/@azure/servicebus/data-plane/test/renewLockSessions.spec.ts

```diff
@@ -71,9 +71,9 @@ describe("Standard", function(): void {
       > {
         await testAutoLockRenewalConfigBehavior(senderClient, receiverClient, {
           maxSessionAutoRenewLockDurationInSeconds: 0,
-          delayBeforeAttemptingToCompleteMessageInSeconds: 31
+          delayBeforeAttemptingToCompleteMessageInSeconds: 31,
+          expectSessionLockLostErrorToBeThrown: true
```

Comment:
Now that we have the error showing up for SessionLockLostError, we should bring back the counterparts to the below from renewLock.spec.ts:

<img width="926" alt="screen shot 2019-02-11 at 11 10 04 am" src="https://user-images.githubusercontent.com/16890566/52586960-adecff00-2ded-11e9-85bb-3d2b793f5cca.png">